packages/botonic-react/src/components/location.jsx

```diff
@@ -31,8 +33,8 @@ export const Location = props => {
   const renderNode = () => {
     return (
       <message type='location'>
-        <lat>{props.lat}</lat>
-        <long>{props.long}</long>
+        <lat>{lat}</lat>
```

Comment:
Is this variable accessible here? I don't think so!

Reply:
i think so, renderNode is inside the scope where props is defined